### ISSUE TITLE
Provide metrics on the usage of the training pages on the PDS web site

### ISSUE THREAD

**Issue by tloubrieu-jpl** [NASA-PDS/pds-site-app#51]
## 💡 Description

Request from Sheri Loftin at the management council face 2 face.

No idea of how that should be done and when that can be done yet.

**Comment by jordanpadams** [comment]
@tloubrieu-jpl good catch. good idea. but I think we should just wait for the new website... this will be so much easier once we can setup google analytics for ourselves
